### DISCUSSION TITLE
Warning on pytest import

### DISCUSSION THREAD

**Discussion by skelutten** [microsoft/pyright#8847] [General]
I get a warning on that:
Import "pytest" could not be resolved
from Pyright.
I have pytest installed in my venv and starting python3 and typing "import pytest" works well.
I have the venv Python as active Python interpreter in my project.
Other Python modules works fine, it is only the Pytest one that is giving me a problem from Pyright.

What could be missing in my config?

Using ms-pyright extension version: v1.1.372

**Comment by erictraut** [comment]
If you set the "verboseOutput" config setting to true, you should see a bunch of additional logging output that may help you diagnose the import resolution issue.
